cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/Users/binh/Documents/workspace/lena-local/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Tools/c_config.py", line 445, in run_c_code\n    bld.compile()\n  File "/Users/binh/Documents/workspace/lena-local/.waf-1.6.11-30618c54883417962c38f5d395f83584/waflib/Build.py", line 190, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 4445692880: cxxprogram test.cpp.1.o -> testprog}\n[\'/usr/bin/g++\', \'test.cpp.1.o\', \'-o\', \'/Users/binh/Documents/workspace/lena-local/build/.conf_check_fc0bab0b855f90bf0795892663948c43/testbuild/testprog\', \'-lrt\']\n'
